plugins/modem-manager/fu-sahara-loader.c

```diff
@@ -118,7 +118,6 @@ sahara_packet_get_length(GByteArray *packet)
 static gboolean
 fu_sahara_loader_find_interface(FuSaharaLoader *self, FuUsbDevice *usb_device, GError **error)
 {
-#ifdef HAVE_GUSB
```

Comment:
I think if making this change it means modem manager needs to explicitly depend on USB support.

Reply:
Ahh indeed, good catch. Fixed.